tomcat8-jsvc@.service

```diff
@@ -1,29 +1,31 @@
 # Systemd unit file for tomcat instances.
 #
 # To create clones of this service:
-# 0. systemctl enable tomcat8-nojsvc@name.service
+# 0. systemctl enable tomcat8@name.service
```

Comment:
I think in the comment, `tomcat8@name.service` should be renamed to `tomcat8-nojsvc@name.service.